src/coreclr/vm/threads.cpp

```diff
@@ -1305,7 +1305,7 @@ Thread::Thread()
 {
     CONTRACTL {
         THROWS;
-        if (GetThreadNULLOk()) {GC_TRIGGERS;} else {DISABLED(GC_NOTRIGGER);}
+        if (GetThreadNULLOk()) {DISABLED(GC_NOTRIGGER);} else {GC_TRIGGERS;}
```

Comment:
```suggestion
        GC_NOTRIGGER;
```
I do not see anything that would trigger GC in the implementation.